chart/develop/values.yaml

```diff
@@ -1,16 +1,12 @@
 nats:
-  cluster:
-    enabled: false
+  replicas: 1
```

Comment:
Thanks, this has been a source of confusion for some people, setting replicas>1 but with cluster false...